packages/near-membrane-base/src/membrane.ts

```diff
@@ -1154,7 +1154,6 @@ export function init(
         return {
             enumerable: isEnumerable,
             configurable: true,
-            writable: true,
```

Comment:
another silly mistake in the code path that is not used atm.

Reply:
Instead of removing it do you want to set writable to `false`?